data/pages/faq-grantees.mdx

```diff
@@ -118,6 +119,11 @@ We encourage you to openly discuss any obstacles that may have prevented you
 from reaching previously established milestones, and/or why your previously
 outlined plans changed. Remember that we're here to help, not to judge.
 
+### What does a poor progress report look like?
+
+- A long wall of text talking about the work but not showing any of the work through relevant links to pull requests and commits.
+- Dives directly and only into a long list of links to pull requests and commits without some contextulization of the work done in those links.
+
 ### Does OpenSats provide feedback on every progress report submitted?
 
 Not necessarily. We review all reports, but this work is mostly done by a team
```

Comment:
I'd suggest:
Not necessarily. Every report is reviewed but we do not provide feedback on every report as we review hundreds each quarter. If you are looking for specific feedback, please call it out as part of the report.

Reply:
[Committ here](https://github.com/OpenSats/website/pull/336/commits/a0067759cf52fff070575c15e578769b6f8e3f6a)